modules/serverpod_auth/serverpod_auth_server/lib/src/business/user_authentication.dart

```diff
@@ -34,21 +45,73 @@ class UserAuthentication {
       method: method,
     );
 
-    session.updateAuthenticated(AuthenticationInfo(userId, scopes));
     var result = await AuthKey.db.insertRow(session, authKey);
+
+    if (updateSession) {
+      session.updateAuthenticated(
+        AuthenticationInfo(
+          userId,
+          scopes,
+          authId: '${result.id}',
+        ),
+      );
+    }
     return result.copyWith(key: key);
   }
 
-  /// Signs out a user from the server and deletes all authentication keys.
-  /// This means that the user will be signed out from all connected devices.
-  static Future<void> signOutUser(Session session, {int? userId}) async {
+  /// Signs out the user from all devices.
+  static Future<void> signOutUser(
+    Session session, {
+    int? userId,
+  }) async {
     userId ??= (await session.authenticated)?.userId;
```

Comment:
Suggestion: Please document this behavior in the methods doumentation.

The old description captured this better. I'm not sure I like this new shorter version more.